src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -556,29 +593,36 @@ impl Net {
                 self.rx_buffer
                     .iovec
                     .write_all_volatile_at(&self.rx_frame_buf[..vnet_hdr_len() + len], 0)?;
-                // SAFETY: This is safe:
+                // SAFETY:
+                // * len will never be bigger that u32::MAX
```

Comment:
nit: this one could use a bit more explanation, e.g. it wont be bigger than 65562 because mmds returns the number of bytes written into `self.rx_frame_buf`, which is an array of size 65562.

Reply:
updated comment